shared/team-building/container.tsx

```diff
@@ -136,6 +136,7 @@ const emptyObj = {}
 
 const mapStateToProps = (state: Container.TypedState, ownProps: OwnProps) => {
   const teamBuildingState = state[ownProps.namespace].teamBuilding
+  const teamBuildingSearchResults = teamBuildingState.teamBuildingSearchResults.toJS()
```

Comment:
call `toJS` in `mergeProps` instead. The output of `mapStateToProps` is compared across state updates to decide whether to render, calling `toJS` here creates a new object each time so this'll render more than it should.

Reply:
Done